### PR TITLE
Email notifications for unreviewed issues

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/91605

I tested it on o3 with
```
echo http://openqa.opensuse.org/tests/2270756 | host=openqa.opensuse.org scheme=http ./openqa-label-known-issues
```
The email looks like:
```
Subject: Unknown issue to be reviewed (Group 24)                                                                    

http://openqa.opensuse.org/tests/2270756 : Unknown issue, to be reviewed ->
http://openqa.opensuse.org/tests/2270756/file/autoinst-log.txt
Reason: null
```

### CODE REVIEW

File: openqa-label-known-issues [hunk 1]
```diff
@@ -17,6 +17,8 @@ issue_marker="${issue_marker:-"auto_review%3A"}"
 issue_query="${issue_query:-"https://progress.opensuse.org/projects/openqav3/issues.json?limit=200&subproject_id=*&subject=~${issue_marker}"}"
 reason_min_length="${reason_min_length:-"8"}"
 grep_timeout="${grep_timeout:-5}"
+email_unreviewed=${email_unreviewed:-true}
+from_email=${from_email:-okurz@suse.de}
```
Comment (okurz) [inline openqa-label-known-issues:21]: Why me? :)

How about

```suggestion
from_email=${from_email:-openqa-label-known-issues@open.qa}
```

which is of course not a valid email address and we should select something infrastructure specfiic wherever we use it

File: test/data/group24.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "exclusively_kept_asset_size": 37640732672,
+    "id": 24,
+    "keep_important_logs_in_days": 365,
+    "keep_important_results_in_days": 3650,
+    "keep_logs_in_days": 60,
+    "keep_results_in_days": 90,
+    "name": "openQA",
+    "parent_id": null,
+    "size_limit_gb": 40,
+    "sort_order": 26
```
Comment (okurz) [inline test/data/group24.json:16]: Please delete all keys that we don't care about to make that test data easier to understand.

**Review by okurz — CHANGES_REQUESTED** [review]

File: openqa-label-known-issues (hunk 1 above)
Comment (perlpunk) [reply openqa-label-known-issues]: done

File: test/data/group24.json (hunk 2 above)
Comment (perlpunk) [reply test/data/group24.json]: done

**Review by Martchus — APPROVED** [review]
Considering the manual test and unit tests I suppose it works :-)
The mails are then supposed to show up in Slack, right?

**Comment by okurz** [comment]
> Considering the manual test and unit tests I suppose it works :-) The mails are then supposed to show up in Slack, right?

That's a possibility, yes. But basically we can try to develop that independantly and leave it to job group owners to decide on which email address the put there as target.

**Review by okurz — APPROVED** [review]

**Comment by perlpunk** [comment]
> The mails are then supposed to show up in Slack, right?

Yes, you should have seen one today at 12:00 in our channel.

**Pull request merged**